### PR TITLE
Add Babel plugin note to Error boundaries post

### PR DESCRIPTION
**Pull request by montogeek**
Following @gaearon recommendation https://twitter.com/spencerbeggs/status/887684609458229253, I added a small note regarding https://www.npmjs.com/package/babel-plugin-transform-react-jsx-source that would help debugging new Error boundaries stack traces.

### CODE REVIEW

**Comment by gaearon** [comment]
I was thinking more like a separate section that would show a screenshot of component traces in development, and then note that for filenames and line numbers you need this transform.

**Comment by gaearon** [comment]
If you’re busy I can do that.

**Comment by montogeek** [comment]
@gaearon I can do it

File: docs/_posts/2017-07-26-error-handling-in-react-16.md [hunk 1]
```diff
@@ -55,7 +55,13 @@ The `componentDidCatch()` method works like a JavaScript `catch {}` block, but f
 
 Note that **error boundaries only catch errors in the components below them in the tree**. An error boundary can’t catch an error within itself. If an error boundary fails trying to render the error message, the error will propagate to the closest error boundary above it. This, too, is similar to how `catch {}` block works in JavaScript.
 
+## Stack traces
+
+By default Error boundaries stack traces would look like this:
```
Comment (gaearon) [inline docs/_posts/2017-07-26-error-handling-in-react-16.md]: Let's add some context here? It's not clear what stack traces this refers to. Something like:

"React 16 prints all errors to the console in development, even if the application accidentally swallows them. In addition to the error message and the JavaScript stack, it also now provides component stacks. Now you can see where exactly in the component tree the failure has happened:"

**Comment by gaearon** [comment]
I tweaked a bit. This looks great. Thank you so much!

**Comment by montogeek** [comment]
Thank you!

**Pull request merged**